system/micd.py

```diff
@@ -3,14 +3,25 @@
 
 from cereal import messaging
 from openpilot.common.realtime import Ratekeeper
+from openpilot.common.retry import retry
 from openpilot.common.swaglog import cloudlog
+from openpilot.system.hardware import TICI
 
 RATE = 10
 FFT_SAMPLES = 4096
 REFERENCE_SPL = 2e-5  # newtons/m^2
 SAMPLE_RATE = 44100
 
 
+@retry(attempts=7, delay=3)
+def wait_for_devices(sd):
```

Comment:
waiting for sounddevice to query more than 1 device seems to also do the trick, but you still have to force reload portaudio